app/src/main/java/org/simple/clinic/editpatient/EditablePatientEntry.kt

```diff
@@ -180,11 +180,14 @@ data class EditablePatientEntry @Deprecated("Use the `from` factory function ins
     when (numberValidator.validate(phoneNumber, Type.LANDLINE_OR_MOBILE)) {
```

Comment:
We need to be more detailed in the commit messages. Instead of something like "Refactor `EditablePatientEntry`", we should be saying exactly what is being done. In this case, it will be something like, "Extract validation logic into discrete methods in `EditablePatientEntry`".